sycl/test-e2e/Regression/kernel_name_class.cpp

```diff
@@ -1,11 +1,7 @@
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
-// RUN: %CPU_RUN_PLACEHOLDER %t.out
-// RUN: %GPU_RUN_PLACEHOLDER %t.out
-// RUN: %ACC_RUN_PLACEHOLDER %t.out
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.ext.out
-// RUN: %CPU_RUN_PLACEHOLDER %t.ext.out
-// RUN: %GPU_RUN_PLACEHOLDER %t.ext.out
-// RUN: %ACC_RUN_PLACEHOLDER %t.ext.out
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// RUN: %{build} -o %t.ext.out
+// RUN: %{run} %t.ext.out
```

Comment:
This is not directly related to this patch and can be addressed separately, but after https://github.com/intel/llvm/pull/9212 we only need the first two lines here

Reply:
https://github.com/intel/llvm/pull/9357